doc/api.html

```diff
@@ -129,6 +129,12 @@ <H2><B>File Open Function</B></H2>
 On Windows, there is also a Windows specific sf_wchar_open() that takes a
 UTF16_BE encoded filename.
 </P>
+<blockquote>
+<p><b>Note:</b></p>
+<p>Starting from 1.0.29 release, behavior of <code>sf_open()</code> on Windows
+ platform was changed, <code>path</code> parameter is treated as UTF-8 string.
+</p>
```

Comment:
As a native English speaker, that test is just a tiny bit off. Can we change it to:
```
From the 1.0.29 release, the behavior of the <code>sf_open()</code>function on Windows
platform has changed, with the <code>path</code> parameter being treated as a UTF-8 string.
```